src/index.ts

```diff
@@ -4,4 +4,42 @@ console.log('hello, world');
 
 const testMessage: string = 'TypeScript works';
 
-console.log(testMessage);
\ No newline at end of file
+console.log(testMessage);
+
```

Comment:
You try to keep the variable range as small as possible. I guess `los`, `liczba`, `ile` can be in `chooseNumber` function